DuckDuckGo/AdAttribution/AdAttributionPixelReporter.swift

```diff
@@ -52,7 +52,11 @@ final class AdAttributionPixelReporter {
             if attributionData.attribution {
                 let parameters = self.pixelParametersForAttribution(attributionData)
                 do {
-                    try await pixelFiring.fire(pixel: .appleAdAttribution, withAdditionalParameters: parameters)
+                    try await pixelFiring.fire(
+                        pixel: .appleAdAttribution,
+                        withAdditionalParameters: parameters,
+                        includedParameters: [.appVersion, .atb]
+                    )
```

Comment:
The attribution feature requires ATB, so I've added it and updated its tests. cc @dus7.

Reply:
Dailan123$&